frontend/src/components/Buttons.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import './Buttons.css';
+
+const Buttons = ({ onUndo, onErase, onToggleNotes, onGetHint }) => {
+  const [notesOn, setNotesOn] = useState(false);
+
+  const toggleNotes = () => {
+    setNotesOn(!notesOn);
+    onToggleNotes(!notesOn); // Call the provided prop function with the updated state
+  };
+
+  return (
+    <div className="buttons-container">
```

Comment:
Continuing from comments on `Buttons.css`, instead of referring to the CSS class, in Tailwind CSS you would include the Tailwind CSS utility class directly into the JS. For example, instead the code here, you would do:

`<div className="p-2.5 text-base cursor-pointer text-white">...`